arbor/fvm_layout.hpp

```diff
@@ -102,6 +102,12 @@ struct fvm_mechanism_config {
     // duplicates for point mechanisms.
     std::vector<index_type> cv;
 
+    // One-to-many map
+    std::vector<index_type> cv_loc;
+
```

Comment:
Which way would you suggest/prefer? 
* If we go with the first what do you think of the name idx_to_mech_params instead of cv_loc? and we could add a comment next to it describing it, something like: maps mechnaism instance index to mechanism properties?

Reply:
With the first way, I'd call it something like `logical_to_mech_index`.

The second approach would be neater though I think: because we have the multiplicity information for each (concrete) mechanism index, these multiplicities could be used to partition the vector of target numbers, e.g. if multiplicity is { 1, 1, 3, 1, 2 } and targets is { 7, 2, 4, 1, 3, 0, 6, 5 }, then we know that target 7 is associated with index 0; target 2 with index 1; targets 4, 1, and 3 with index 2; and so on.